src/app/views/home/Home.js

```diff
@@ -159,8 +159,9 @@ export default class Home extends React.Component {
                   <h4 id="date">May 20 - 22, 2022</h4>
                   <p id="tagline">UC Irvine's largest women-centric hackathon</p>
                     <Button id="apply-btn">
-                      <a href="https://venushacks-2021.devpost.com/" style={{ color: "#99a6e5"}}>
-                        DEVPOST
+                      <a href="https://airtable.com/shr1Etr4Typ3FEqBc" target="_blank" 
+                      style={{ color: "#99a6e5"}}>
```

Comment:
- Add the style to the css file
- Add a noreferrer to the a tag (there should be a warning about this in the terminal)

Reply:
Just committed these changes, I also used <rel="noopener noreferrer"> rather than just <rel="noreferrer"> which is hopefully okay